modules/packages/TOML.chpl

```diff
@@ -919,8 +919,8 @@ used to recursively hold tables and respective values
     pragma "no doc"
     /* Send values from table to toString for writing  */
     proc printValues(f: channel, v: borrowed Toml) throws {
-      for (key, val) in v.A.items() {
-        var value = val!;
+      for key in v.A.keysToArray().sorted() {
```

Comment:
since we're talking about deprecating `array.sorted()` it might be good to write `var keys = v.A.keysToArray(); sort(keys)`, likewise below.